src/styles/index.scss

```diff
@@ -1,3 +1,103 @@
 body {
   margin: 0;
+  color: #060b35;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-wrap: wrap;
+  width: 200px;
+
+  &__info {
+    margin: 36px 16px 16px;
+    font-size: 12px;
+    line-height: 18px;
+  }
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    &-block {
+      margin: 32px 19px 0;
+    }
+  }
+
+  &__product-code {
+    text-align: left;
+    color: #616070;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+  &__stars {
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+  &__price-block {
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    display: flex;
+  }
+  &__price-text {
+    color: #616070;
+    font-size: 12px;
+    font-weight: 400;
+  }
+  &__price-numbers {
+    font-size: 16px;
+    font-weight: 700;
+  }
+  &__button {
+    text-transform: uppercase;
+    text-align: center;
+    color: #fff;
+    background-color: #00acdc;
+    border: none;
+    border-radius: 5px;
+    width: 100%;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    &:hover {
+      cursor: pointer;
+      color: #00acdc;
+      background-color: #fff;
```

Comment:
Consider using variables for colors to maintain consistency and make it easier to update the color scheme in the future. This will also improve the maintainability of your code.